Gemfile.lock

```diff
@@ -417,20 +417,20 @@ GEM
       coercible (~> 1.0)
       descendants_tracker (~> 0.0, >= 0.0.3)
       equalizer (~> 0.0, >= 0.0.9)
-    warden (1.2.8)
-      rack (>= 2.0.6)
+    warden (1.2.9)
+      rack (>= 2.0.9)
     webmock (2.3.2)
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (4.2.2)
+    webpacker (4.3.0)
```

Comment:
### [webpacker](https://github.com/rails/webpacker)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.2.2 → 4.3.0

[_[change-log](https://github.com/rails/webpacker/blob/v4.3.0/CHANGELOG.md), [source-code](https://github.com/rails/webpacker/tree/v4.3.0)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/rails/webpacker/compare/v4.2.2...v4.3.0).

These are the individual commits:
- (e8e0910) [bump compression-webpack-plugin to ^4.0.0 (#2695)](https://github.com/rails/webpacker/commit/e8e09101dd6d4e665ae24c36c81e0cc5aaf29fc6)
- (afb1f26) [4.3.0](https://github.com/rails/webpacker/commit/afb1f263e3dbbcdbe09bd7f53dd763536c37f6d9)


</details>